tchannel/tornado/tchannel.py

```diff
@@ -212,6 +213,12 @@ def request(self,
 
             Defaults to 'c'.
         """
+        # TODO: do not warn if being called from tchannel.TChannel
```

Comment:
I was thinking: We make `call` implemented in terms of `request.send` available, release that, then we invert refactor and invert it so that `request.send` is implemented in terms of `call`, and that's when we add the deprecated warning to `call`. We release that. Then we release a new version that doesn't have `request.send`.